src/main/java/com/getbase/models/User.java

```diff
@@ -113,6 +155,107 @@ public int hashCode() {
     result = 31 * result + (email != null ? email.hashCode() : 0);
     result = 31 * result + (name != null ? name.hashCode() : 0);
     result = 31 * result + (deletedAt != null ? deletedAt.hashCode() : 0);
+    result = 31 * result + (reportsTo != null ? reportsTo.hashCode() : 0);
+    result = 31 * result + (timezone != null ? timezone.hashCode() : 0);
+    result = 31 * result + (group != null ? group.hashCode() : 0);
+    result = 31 * result + roles.hashCode();
```

Comment:
why `phoneNumber` and `teamName` are skipped here?

Reply:
by mistake -- fixed.